.changeset/big-spiders-develop.md

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
![image](https://github.com/user-attachments/assets/f4c0c6b2-3de5-438b-84fd-94d89ee7c7f8)
![image](https://github.com/user-attachments/assets/98e04ce8-a0da-4245-b8b7-29f316044426)

Where are the disabled colors coming from? When an has a border and is disabled we usually ignore the border, see TextField Disabled state.

Reply:
just using the disabled tokens. The disabled token for border is the same color as the surface. There is still a border for TextField and Button for disabled states.
Oh I see now, it's because disclosure has a box-shadow. That's why it's a bit different... It wasn't int he design, so not sure if we want to remove this when disabled?

Reply:
I fixed border-color and removed box-shadow when disabled.

Reply:
I think it's the way to go since it is not interactive therefore the box-shadow is not useful